programs/ziti-prox-c/proxy.c

```diff
@@ -311,9 +312,18 @@ static void binding_listen_cb(ziti_connection srv, int status) {
 }
 
 static void on_ziti_accept(ziti_connection clt, int status) {
-    uv_stream_t *s = ziti_conn_data(clt);
+    uv_handle_t *s = ziti_conn_data(clt);
     if (status == ZITI_OK) {
-        ziti_conn_bridge(clt, s, on_bridge_close);
+        if (ziti_conn_bridge(clt, s, on_bridge_close) == 0) {
+            if (s->type == UV_UDP) {
+                ziti_conn_bridge_idle_timeout(clt, 10000);
```

Comment:
magic number `10000`? Should be a constant or configurable

Reply:
technically it is a literal constant